cloudhsm/base-images/jdk11-jce-gradle/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Base builder image to build apps that need the HSM
+
+ARG JAVA_HOME=/opt/java/openjdk
+ARG GRADLE_HOME=/opt/gradle
+FROM gradle:jdk11 AS build
+
+FROM amazonlinux:2.0.20190508
+
+# Install Gradle and Java
+ARG JAVA_HOME
```

Comment:
Might be my lack of understanding of Dockerfiles but do you need lines 10 and 11 here considering these have already been defined on lines 3 and 4?

Reply:
Yep - you need to redefine the `ARG`s at every build stage as they're not propagated